dvol.go

```diff
@@ -0,0 +1,16 @@
+package main
+
+import (
+    "fmt"
+
+//    "github.com/clusterhq/dvol/dockercontainers"
+//    "github.com/clusterhq/dvol/plugin"
```

Comment:
I recommend deleting all of the Go stuff from this change and just focusing on the out-of-process stuff. The Go bits can be in the next patch.